plugins/wasmembed/secure-enclave.cpp

```diff
@@ -266,29 +267,22 @@ class WasmEngine
         return found->second.data(store.context());
     }
 };
-static std::unique_ptr<WasmEngine> wasmEngine;
+thread_local std::unique_ptr<WasmStore> wasmStore = std::make_unique<WasmStore>();
```

Comment:
Potential problems with code like this on osx and efficiency of thread_local variables with non-trivial constructors.

Reply:
I am not sure how the lifetime of threading works within thor jobs - but I suspect this must be more efficient than a lock on the call.

Reply:
Open new ticket for efficiency access to wasm store:  Move `std::make_unique<WasmStore>();` into secure function constructor (testing if it exists first).